tests/unit/conftest.py

```diff
@@ -43,8 +42,8 @@ def clean():
 @pytest.fixture(scope="module")
 def credentials():
     return GlueCredentials(
-        role_arn=f"arn:aws:iam::{account}:role/GlueInteractiveSessionRole",
-        region=region,
+        role_arn=get_role_arn(),
+        region=get_region(),
```

Comment:
Would you update the `session_provisioning_timeout_in_seconds` to 60 or 120 seconds? 30 second is a bit short and it sometimes timeout.

Reply:
fixed